pages/pets/show.tsx

```diff
@@ -2,25 +2,25 @@ import React, { useEffect, useState } from 'react'
 import axios from 'axios';
 import update from "immutability-helper";
 
-interface GamesShowProps {
+interface PetsShowProps {
   id?: string
 }
 
-const GamesShow:React.FC<GamesShowProps> = () => {
-  const [game, setGame] = useState<any>('')
+const PetsShow:React.FC<PetsShowProps> = () => {
+  const [pet, setPet] = useState<any>('')
 
-  const loadGame = () => {
+  const loadPet = () => {
     axios
-      .get("http://localhost:3001/api/v1/games")
+      .get("http://localhost:3001/api/v1/pet")
```

Comment:
This url will need to match the route in the rails/routes file, and will also need to pass in the ID like this: 
```suggestion
      .get(`http://localhost:3001/api/v1/pet/${props.id}`)
```